### PR TITLE
Feature/15 controller location service

### PR DESCRIPTION
**Pull request by thejaneshin**
Requesting pull request to merge feature/15-controller-locationService to development.

### CODE REVIEW

File: log.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+2019-12-01 20:35:10 TRACE root:29 - Inside getAllTrainingLocation Service method.
```
Comment (jurczakn) [inline log.txt]: remove logs

File: src/test/java/com/revature/test/HousingLocationServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  }
+
+  @Test
+  @Sql("housing-location-script.sql")
+  void testGetAllHousingLocation() {
+    List<HousingLocation> existingHLocationList = new ArrayList<>();
+    existingHLocationList.add(existingHousingLocation);
+    existingHLocationList.add(updatedHousingLocation);
+    assertEquals(housingLocationServiceImpl.getAllHousingLocations(), existingHLocationList);
+    System.out.println(housingLocationServiceImpl.getAllHousingLocations());
```
Comment (jurczakn) [inline src/test/java/com/revature/test/HousingLocationServiceImplIntegrationTest.java]: remove sysout

File: src/test/java/com/revature/test/TrainingLocationServiceImplIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  @Test
+  @Sql("training-location-script.sql")
+  void testGetAllTrainingLocation() {
+    List<TrainingLocation> existingTLocationList = new ArrayList<>();
+    existingTLocationList.add(existingTrainingLocation);
+    existingTLocationList.add(updatedTrainingLocation);
+    assertEquals(trainingLocationServiceImpl.getAllTrainingLocations(), existingTLocationList);
+
+    System.out.println(trainingLocationServiceImpl.getAllTrainingLocations());
```
Comment (jurczakn) [inline src/test/java/com/revature/test/TrainingLocationServiceImplIntegrationTest.java]: remove sysout

File: log.txt (hunk 1 above)
Comment (Astralvix) [reply log.txt]: all set

File: src/test/java/com/revature/test/HousingLocationServiceImplIntegrationTest.java (hunk 2 above)
Comment (Astralvix) [reply src/test/java/com/revature/test/HousingLocationServiceImplIntegrationTest.java]: all set

File: src/test/java/com/revature/test/TrainingLocationServiceImplIntegrationTest.java (hunk 3 above)
Comment (Astralvix) [reply src/test/java/com/revature/test/TrainingLocationServiceImplIntegrationTest.java]: all set

**Review by jurczakn — APPROVED** [review]
good

**Pull request merged**